spec/features/us01_signup_spec.rb

```diff
@@ -0,0 +1,69 @@
+include Support::Auth
+
+feature 'Sign up', type: :feature, js: true do
+
+  scenario 'when user have valid credentials' do
+    sign_up attributes_for(:user)
+    expect(page).to have_content 'You’re successfully registered!'
+    sleep 1
```

Comment:
You should avoid using sleep in your tests because it means that your app works too slow and time of sleep depends on performance the machine where the tests are run.

Reply:
removed